### PR TITLE
SEO-171884-Page-wiith-redirection-fix-5

### PR DESCRIPTION
**Pull request by Thahira-Banu**
Hi @Aishwarya-Ganesan 
  I had made changes in page with redirection website links by changing image name in md file and image folder.

Thanks & Regards
Thahira Banu

### CODE REVIEW

File: uwp/DateTimePicker/Overview.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ documentation: ug
 
 The SfDateTimeCombo control displays each part of a date or time in a separate DateTimeItem drop-down list that can be edited. This control can be used to manipulate DateTime objects, using a familiar drop-down list in a touch-friendly manner.
 
-![DateTimeCombo displayed date and time with dropdown option to select date and time](Overview_images/Overview_img1.png)
+![DateTimeCombo displayed date and time with dropdown option to select date and time](uwp-datepicker-overview.png)
```
Comment (MallikaRK) [inline uwp/DateTimePicker/Overview.md:14]: where is image folder?

File: uwp/Diagram/Getting-Started.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ documentation: ug
 
 # Getting Started with UWP Diagram (SfDiagram)
 
-The following section helps you to build your application with SfDiagram. 
+The following saection helps you to build your application with SfDiagram.
```
Comment (MallikaRK) [inline uwp/Diagram/Getting-Started.md:12]: Why change the spelling here? revert it

**Review by MallikaRK — CHANGES_REQUESTED** [review]
Commented

File: uwp/Diagram/Getting-Started.md (hunk 2 above)
Comment (Thahira-Banu) [reply uwp/Diagram/Getting-Started.md:12]: resolved

File: uwp/Diagram/Shapes.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: post
-title: Shapes in UWP Diagram control | Syncfusion
+title: Shapes in UWP Diagram uwp-diagram-shapesontrol | Syncfusion
```
Comment (Aishwarya-Ganesan) [inline uwp/Diagram/Shapes.md:3]: why title changed like this?

File: uwp/Docking/Getting-Started.md [hunk 4]
```diff
@@ -111,7 +111,7 @@ xmlns:layout="using:Syncfusion.UI.Xaml.Controls.Layout">
 
 {% endtabs %}
 
-![Dock windows with header in DockingManager](Getting-Started-images/Getting-Started-img2.jpeg)
+![Dock windows with header in UWP Docking Manager](getting-started-images/uwp-diagram-set-header.jpeg)
```
Comment (Aishwarya-Ganesan) [inline uwp/Docking/Getting-Started.md:114]: why uwp diagram added here?

File: uwp/Diagram/Shapes.md (hunk 3 above)
Comment (Thahira-Banu) [reply uwp/Diagram/Shapes.md:3]: resolved

File: uwp/Hub-Tile/SfHubTile.md [hunk 5]
```diff
@@ -345,24 +345,24 @@ hubTile.Interval = TimeSpan.FromSeconds(1)
...
 
-![Hubtile rotate content vertically](SfHubTile-images/SfHubTile-img7.jpeg)
+![UWP SFHubTile rotate content vertically](sfhubtile-images/uwp-hubtile-rotate-content-vertically.jpeg)
 
 ### Tile slide animation
 
 Main and Secondary contents slide with specified interval.
 
-![Hubtile side content with animation](SfHubTile-images/SfHubTile-img8.jpeg)
+![UWP SFHubTile side content with animation](SfHubTile-images/uwp-hubtile-side-content-with-animation.jpeg)
```
Comment (Aishwarya-Ganesan) [inline uwp/Hub-Tile/SfHubTile.md:356]: still capital letters there

**Review by Aishwarya-Ganesan — APPROVED** [review]
Changes are fine.

**Review by Aishwarya-Ganesan — APPROVED** [review]
Changes are fine.

File: uwp/Diagram/Shapes.md [hunk 6]
```diff
@@ -42,19 +42,19 @@ The following code example illustrates how to assign Shape property of the Node.
...
  
 The list of flow shapes are as follows
 
-![UWP SfDiagram Shapes Image4](Shapes_images\Shapes_image4.PNG)
+![UWP SfDiagram Flow Shapes](shapes-images\uwp-diagram-flow-shapes.PNG)
  
 The list of arrow shapes are as follows
 
-![UWP SfDiagram Shapes Image5](Shapes_images\Shapes_image5.PNG)
+![UWP SfDiagram Arrow Shapes](shapes-images\uwp-diagaram-arrow-shapes.PNG)
```
Comment (MallikaRK) [inline uwp/Diagram/Shapes.md:57]: Check diagram spelling in image name.

File: uwp/Hub-Tile/SfpulsingTile.md [hunk 7]
```diff
@@ -116,7 +116,7 @@ Set the image as Content of `SfPulsingTile` as given below:
 
 {% endtabs %}
 
-![Pulsing tile displayed bitmap](SfpulsingTile-images/SfpulsingTile-img1.jpeg)
+![UWP SFPulsing tile displayed bitmap](sfpulsingtile-images/uwp-hub-tile-adding-image-to-sfpulsingtile.jpeg)
```
Comment (MallikaRK) [inline uwp/Hub-Tile/SfpulsingTile.md:119]: double time mentioned control name in image name. 
Pulsing tile is the control name here. So, change like as below.
adding-images-in-uwp-pulsing-tile

File: uwp/Hub-Tile/SfMosaicTile.md [hunk 8]
```diff
@@ -108,7 +108,7 @@ Set the properties Header, ImageList as given below:
 
 {% endtabs %}
 
-![Hubtile displayed collection images](SfMosaicTile-images/SfMosaicTile-img1.jpeg)
+![Add image list in UWP SFMosaicTile](sfmosaictile-images/uwp-hub-tile-add-image-list-in-sfmosaictile.jpeg)
```
Comment (MallikaRK) [inline uwp/Hub-Tile/SfMosaicTile.md:111]: remove uwp-hub-tile in image name. 
mosaictile is the control name,
add-image-list-in-uwp-sfmosaictile

Change the other image name as per the above.

**Review by MallikaRK — CHANGES_REQUESTED** [review]
Commented

File: uwp/Hub-Tile/SfpulsingTile.md (hunk 7 above)
Comment (Thahira-Banu) [reply uwp/Hub-Tile/SfpulsingTile.md:119]: resolved

**Review by MallikaRK — APPROVED** [review]
Approved

**Pull request merged**